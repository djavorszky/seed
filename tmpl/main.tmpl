package main

import (
{{ range .Imports }}    "{{.}}"
{{end}}
)

func main() {
	service := gen.New(&{{.ServicePackage}}.Server{})

	log.Fatal(http.ListenAndServe("{{.ServiceAddress}}", service))
}
