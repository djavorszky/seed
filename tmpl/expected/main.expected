package main

import (
	"admiral"
	"admiral/gen"
	"log"
	"net/http"
)

func main() {
	service := gen.New(&admiral.Server{})

	log.Fatal(http.ListenAndServe(":8080", service))
}
